pkgs/development/compilers/dotnet/sdk/8.0/default.nix

```
@@ -0,0 +1,27 @@
+{ callPackage
+, dotnetCorePackages
+, stdenvNoCC
+, fetchurl
+}: callPackage ../generic.nix {} {
+  releaseManifest = ./release.json;
+  sha256 = "sha256-pLtgOuFacpKpsltspY5Eza0ePcKjPcuM/TROAIh+fbI=";
+  prepDepsFile = ./prep-deps.nix;
+  buildDepsFile = ./build-deps.nix;
+  bootstrapSdk = dotnetCorePackages.sdk_8_0 // {
+    artifacts = stdenvNoCC.mkDerivation rec {
+      name = "Private.SourceBuilt.Artifacts.8.0.100-rc.2.23502.1.centos.8-x64";
```

Does this need to be updated to the RTM version?

```suggestion
      name = "Private.SourceBuilt.Artifacts.8.0.100-rtm.23551.1.centos.8-x64";
```

I know the `v8.0.0` tag has this version[^1], but the `release/v8.0.1xx` branch uses the RTM version[^2]. I also get a warning when viewing the `v8.0.0` tag:

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

[^1]: https://github.com/dotnet/dotnet/blob/v8.0.0/eng/Versions.props#L33
[^2]: https://github.com/dotnet/dotnet/blob/release/8.0.1xx/eng/Versions.props#L33

---

It was meant to match what's in the repo [1].  In theory we could use our own build of this if we had previous versions packaged.